### ISSUE TITLE
separate tutorial pages from the code

### ISSUE THREAD

**Issue by jo-pol** [d-bl/GroundForge#163]
Each repository has one license. Code licenses are from another family than documentation licenses. So the tutorial pages should move to their own repository. 

**Practical notes**:

* The landing page should stay here
* [x] The new repository name suggested so far: `groundforge-help`
* [x] Some images have to move from docs/help/images to docs/images as they are used on the active page.
* [x] Content of not yet implemented pop-ups may need to stay here
* [x] To keep the original history see
  https://github.com/d-bl/gw-lace-to-gf/blob/master/pruned.md
  https://github.com/d-bl/tesselace-to-gf/blob/master/pruned.md
* [x] change license of the new repository
* [x] reroute links from other repositories to the new one
* [x] drop the help pages from the original repository, add a hint to the 404 page
* [x] check for broken links

**Comment by veronika** [comment]
How about d-bl.github.io/gf-help?

**Comment by jo-pol** [comment]
Or `help-for-gf` to keep the uniform tail `-gf` as for the other satellite repositories.

**Comment by jo-pol** [comment]
Perhaps `GroundForge-help` is a good solution. Then only the `/` changes into a `-`. 

For technical reasons the [landing page](https://d-bl.github.io/GroundForge/) should stay put. So the next challenge: what to do with its sidebar? Keeping it in sync with the help repository would complicate maintenance too much. Returning to the original sidebar of the [github architect theme](https://pages-themes.github.io/architect/).

**Comment by jo-pol** [comment]
Some images have to move from `docs/help/images` to `docs/images` as they aye used on the active page.

**Comment by jo-pol** [comment]
@d-bl/gf 

As of coming weekend I intend to take a deep breath to separate the help pages into another repository and change their licenses.
* Should you have offline pending changes, please submit them before or it might get complicated.
* Two help-like pages will have to stay for technical reasons, the [landing page](https://jo-pol.github.io/GroundForge/) and the [404](https://jo-pol.github.io/GroundForge/404). I don't like the prospect of keeping the sidebar in sync in two repositories. On the other hand the sidebar on the 404 page can be convenient for visitors to recover broken links, so what to with it?

**Comment by jo-pol** [comment]
The required changes are pushed to https://jo-pol.github.io/GroundForge together with MAG-gf, gw-lace-to-gf and tesselace-to-gf. `deadlinkchecker.com` only complains about `jo-pol.github.io` that is OK. That page indeed does not exist but `d-bl.github.io` does exist.

This checker "only" checks the first 2000 links found on a domain. Most patterns (144 for gw-lace-to-gf, over 650 for tesselace-to-gf, ... for MAE-gf) have an image and multiple links so that quickly exceeds the limits. Other checkers I found follow links on a page until a certain depth but also complain about too many links. I used to check the generated pages automatically. That was too buggy, so I [dropped](https://github.com/jo-pol/GroundForge/commit/42545a9b4fc36475414ad12013531edc688920dc) it. I thought I found another reasonably simple [custom solution](https://blog.postman.com/check-for-broken-links-on-your-website-using-a-postman-collection/) but I seem to miss steps or elements.  
Anyone a suggestion?
